src/System.ValueTuple/tests/ValueTupleTests.cs

```diff
@@ -1225,6 +1225,107 @@ public static void EightTuplesWithBadRest()
 #endif
         }
 
+#if netcoreapp11
+        [Fact]
+#endif
+        public static void ValueTupleSerialization()
+        {
+            VerifySerialization(new ValueTuple(),
```

Comment:
Indeed the binary serialization fails (because the reference package from CoreCLR doesn't have the attribute yet). I'll wait for the reference version to be updated.